Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -143,6 +143,9 @@
 //import com.facebook.LoggingBehavior;
 //import com.facebook.applinks.AppLinkData;
 
+import com.facebook.FacebookSdk;
```

Comment:
@PraveenGongada Can you remove imports if it is not used

Reply:
This import will be used internally